src/Microsoft.Security.Utilities.Core/Microsoft.Security.Utilities.Core.csproj

```diff
@@ -18,6 +18,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
>PropertyGroup [](http://example.com/codeflow?start=3&length=13)

shouldn't we put this in build.props? it seems more likely to be picked up by upstream builds...

Reply:
@michaelcfanning That makes sense. Good point.

Reply:
for my understanding this is to change format form `default` to `portable`
not `Adding portable pdb generation support`
the `default` is different for the .net version used.
Do we think the default was `Full` ?

Reply:
@shaopeng-gh Per [this](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation), the default format does seem to be `full`.

Reply:
for vscode wiki it says: `With .NET Core "SDK"-style .csproj's, Portable PDBs are already enabled by default` 
[https://github.com/dotnet/vscode-csharp/wiki/Portable-PDBs](https://github.com/dotnet/vscode-csharp/wiki/Portable-PDBs)

// First four bytes should be 'BSJB'
[https://github.com/microsoft/vstest/blob/80af388e851d0d8b88f93d07d46d47054acb9951/src/Microsoft.TestPlatform.ObjectModel/Navigation/PortablePdbReader.cs#L106](https://github.com/microsoft/vstest/blob/80af388e851d0d8b88f93d07d46d47054acb9951/src/Microsoft.TestPlatform.ObjectModel/Navigation/PortablePdbReader.cs#L106)

it looks like already is:

Reply:
@shaopeng-gh This is in response to the following error message I received while publishing the package to Nuget:

![image](https://github.com/user-attachments/assets/0cd2e0d6-e868-46d0-91e5-61354b7a30a7)

This is why I am trying out setting the `portable` DebugType explicitly. You have reasons to believe this won't help?

Reply:
The one that is not with `BSJB` is the  lower case one. 
Is that a ref, ref will not get changed by the settings in current repo.

![image](https://github.com/user-attachments/assets/2063ca0a-75e9-4742-9add-92e98c6b9401)